commands/python/command/cluster_init.py

```diff
@@ -18,15 +18,20 @@ def parse_args(sub_parser):
         action="store_true",
         help="don't check if cluster init projects are already installed before installing",
     )
+    subparser.add_argument(
```

Comment:
Support helm2 options on aladdin commands during migration in case something fails to move over